generators/app/templates/github/workflows/validate.yml

```diff
@@ -9,11 +9,10 @@ on:
       - 'beta'
       - 'alpha'
       - '!all-contributors/**'
-  pull_request: {}
+  pull_request:
+    branches-ignore: ['all-contributors/**']
```

Comment:
```suggestion
  pull_request: {}
```

Reply:
Is there a reason the other approach is better? I think this looks cleaner as it uses the same syntax for both branches and pull requests.

Reply:
The way you did it wasn't working, that's why @kentcdodds changed it back and added the `if` statement